org.tropy.Tropy.yml

```diff
@@ -0,0 +1,59 @@
+app-id: org.tropy.Tropy
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: tropy
+
+finish-args:
+  - --share=ipc
+  #- --socket=wayland
+  #- --socket=fallback-x11
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-public-share
+  - --filesystem=xdg-run/gvfsd
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.indicator.application
+  - --talk-name=com.canonical.Unity.LauncherEntry
+
+modules:
+  - name: tropy
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/tropy/tropy/releases/download/v1.14.0/tropy-1.14.0-x64.tar.bz2
```

Comment:
does this include fixes for the webp vulnerability?

Reply:
Uses 24.2.0 https://github.com/tropy/tropy/blob/0afaddaddbbc60ea2982afc5331981f53d8e89c8/package.json#L68C19-L68C25, 24.8.3 is the fixed version is v24 branch afaik.